lib/anchors.js

```diff
@@ -20,11 +20,7 @@ const anchor = function(md, opts) {
   const originalHeadingOpen = md.renderer.rules.heading_open;
 
   md.renderer.rules.heading_open = function(tokens, idx, something, somethingelse, self) {
-    let args = [];
-
-    for (let _len = arguments.length, _key = 0; _key < _len; _key++) {
-      args.push(arguments[_key]);
-    }
+    const args = [...arguments];
```

Comment:
Thank you for correcting the review.
But the intention of the review was actually different. I was talking about the rest parameter, not the spread syntax. Your corrections were corrected by reading the review text correctly. Thank you, and I apologize.
Change code from `const args = [... arguments]` to `const [tokens, idx, something, something else, self] = args` and the dummy argument itself to `(... args)` (does not generate the arguments object itself).
By making changes, your code will be more like es2015.

Reply:
Thanks for the guidance.